packages/twenty-server/src/database/typeorm/core/migrations/1709680520888-dropAndCreateUserWorkspace.ts

```diff
@@ -0,0 +1,40 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class DropAndCreateUserWorkspace1709680520888
+  implements MigrationInterface
+{
+  name = 'DropAndCreateUserWorkspace1709680520888';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            DROP TABLE "core"."userWorkspace";
```

Comment:
This seems very dangerous? Why are you dropping the whole table - I don't really get it?

Reply:
Like we discussed on discord, I have put up a migration to drop and add the constraints along with a warning comment!